.github/workflows/release.yml

```diff
@@ -18,14 +18,13 @@ jobs:
       package: ${{ steps.regex-match.outputs.group1 }}
       
     steps:
-      - name: Extract semver ℹ️
+      - name: Validate tag ℹ️
```

Comment:
What does it mean `i` ? :thinking:

Reply:
It was already there.

Reply:
I guess it was a leftover. I recommend to clean it up, eventually